--- a/libappmonitor/Makefile.am	2026-04-08 10:44:40.783080395 +0300
+++ a/libappmonitor/Makefile.am.orig	2026-04-08 10:44:49.827742334 +0300
@@ -42,7 +42,6 @@ 
 libappmonitor_la_SOURCES += $(top_srcdir)/lib/stubs/stub-panic.c
 libappmonitor_la_SOURCES += $(libappmonitor_rpcchanneldir)/bdoorChannel.c
 libappmonitor_la_SOURCES += $(libappmonitor_rpcchanneldir)/rpcChannel.c
-libappmonitor_la_SOURCES += $(libappmonitor_rpcchanneldir)/glib_stubs.c
 if HAVE_VSOCK
 libappmonitor_la_SOURCES += $(libappmonitor_rpcchanneldir)/vsockChannel.c
 libappmonitor_la_SOURCES += $(libappmonitor_rpcchanneldir)/simpleSocket.c
--- a/libguestlib/Makefile.am	2026-04-08 13:40:16.510875436 +0300
+++ a/libguestlib/Makefile.am.orig	2026-04-08 13:40:38.187062087 +0300
@@ -44,7 +44,6 @@ 
 libguestlib_la_SOURCES += vmGuestLib.c
 libguestlib_la_SOURCES += $(libguestlib_rpcchanneldir)/bdoorChannel.c
 libguestlib_la_SOURCES += $(libguestlib_rpcchanneldir)/rpcChannel.c
-libguestlib_la_SOURCES += $(libguestlib_rpcchanneldir)/glib_stubs.c
 if HAVE_VSOCK
 libguestlib_la_SOURCES += $(libguestlib_rpcchanneldir)/vsockChannel.c
 libguestlib_la_SOURCES += $(libguestlib_rpcchanneldir)/simpleSocket.c
@@ -55,6 +54,8 @@ 
 # Needed for OS's that don't link shared libraries against libc by default, e.g. FreeBSD
 libguestlib_la_LIBADD += -lc
 
+libguestlib_la_LIBADD += @GLIB2_LIBS@
+
 libguestlib_includedir = $(includedir)/vmGuestLib
 
 libguestlib_include_HEADERS =
